src/main/java/com/yello/server/domain/user/entity/Social.java

```diff
@@ -1,15 +1,25 @@
 package com.yello.server.domain.user.entity;
 
+import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
+import java.util.Arrays;
+@Getter
 @RequiredArgsConstructor
 public enum Social {
     KAKAO("Kakao"),
     APPLE("Apple");
 
-    private final String social;
+    private final String intial;
 
-    public String social() {
-        return social;
+    public String intial() {
+        return intial;
+    }
+
+    public static Social fromCode(String dbData) {
+        return Arrays.stream(Social.values())
+                .filter(v -> v.getIntial().equals(dbData))
+                .findAny()
+                .orElseThrow(() -> new IllegalArgumentException(String.format("존재하지 않는 성별입니다.", dbData)));
```

Comment:
요거 제 코드 머지해도 사용할 메서드인가요ㅕ??
메세지가 성별로 되어있는 것 같아서 수정이 필요해보입니다.
```suggestion
                .orElseThrow(() -> new IllegalArgumentException(String.format("존재하지 않는 소셜입니다.", dbData)));
```

Reply:
넴 enumConverter 때문에 사용했습니다~